lib/fruitstrap.js

```diff
@@ -0,0 +1,52 @@
+import { exec } from 'teen_process';
+import logger from './logger';
+
+const FRUITSAP_PATH = 'fruitstrap';
+
+class Fruitstrap {
+ 
+  constructor (udid, opts = {}) {
+    this.udid = udid;
+    this.fruitstrapPath = opts.path || FRUITSAP_PATH;
+  }
+
+  async remove (bundleid) {
+    let remove = [`uninstall`, `--id`, `${this.udid}`, `--bundle`, `${bundleid}`];
+    try {
+      let {stdout} = await exec(`${this.fruitstrapPath}`, remove, { maxBuffer: 524288});
```

Comment:
No need for `${this.fruitstrapPath}`, since `this.fruitstrapPath` should already be a string.